MobiFlight/ExecutionManager.cs

```diff
@@ -918,6 +925,11 @@ private ConnectorValue ExecuteRead(OutputConfigItem cfg)
                 result.type = FSUIPCOffsetType.Float;
                 result.Float64 = xplaneCache.readDataRef(source.XplaneDataRef.Path);
             }
+            else if (cfg.Source is ProSimSource)
+            {
+                var source = cfg.Source as ProSimSource;
+                result = ProSim.ProSimHelper.executeRead(source.ProSimDataRef.Path, proSimCache);

```

Comment:
Will probably simplify this to not a "Helper" layer, as it is not needed. Remnant from following existing pattern